packages/@okta/vuepress-site/docs/guides/create-an-api-token/main/index.md

```diff
@@ -0,0 +1,76 @@
+
+```http
+Authorization: SSWS 00QCjAl4MlV-WPXM...0HmjFx-vbGua
+```
+
+### Privilege level
+
+Different Okta API operations require different admin privilege levels. API tokens inherit the privilege level of the admin account that is used to create them. It is therefore good practice to create a service account to use when you create API tokens so that you can assign the token the specific privilege level needed. See [Administrators](https://help.okta.com/okta_help.htm?id=ext_Security_Administrators) for admin account types and the specific privileges of each.
+
+### OAuth 2.0 instead of API tokens
+
+As an alternative to Okta API tokens, you can now interact with Okta APIs using scoped OAuth 2.0 access tokens for a number of Okta endpoints. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains. For more information, see the [OAuth for Okta](/docs/guides/implement-oauth-for-okta/) guide.
+
+## Create the token
+
+To create an API token, follow these steps:
+
+1. Sign in to your Okta organization as a user with [administrator privileges](https://help.okta.com/okta_help.htm?id=ext_Security_Administrators).
+
+     API tokens have the same permissions as the user who creates them, and if the user permissions change, the API token permissions also change.
+
+    See the note above on <GuideLink link="../overview/#privilege-level">Privilege level</GuideLink>, regarding the use of a service account when creating an API token, to specifically control the privilege level associated with the token.
+
+    If you don't have an Okta organization, you can [create one for free](https://developer.okta.com/signup).
+
+2. Access the API page: In the Admin Console, select **API** from the **Security** menu and then select the **Tokens** tab.
+
+3. Click **Create Token**.
+
+4. Name your token and click **Create Token**.
+
+5. Record the token value. This is the only opportunity to see it and record it.
+
+## Token expiration and deactivation
+
+Tokens expire automatically after a certain period and can also be deactivated at any time.
+
+### Token expiration
+
+Tokens are valid for 30 days from creation or last use, so that the 30 day expiration automatically refreshes with each API call. Tokens that aren't used for 30 days expire. The 30-day period is currently fixed and can't be changed for your organization.
+
+### Token deactivation
+
+If a user account is deactivated in Okta, any API token created by that user account is deprovisioned at the same time.
+
+## Next steps
+
+With the token created, you can begin using it, supplying it in the `Authorization header` of calls to Okta API endpoints.
+
+## See also
```

Comment:
These "See also" links are perfect; just the kind of related aside content I was envisaging.